content/en/platform/corda/5.0-dev-preview-2/getting-started/first-flow/code-java.md

```diff
@@ -0,0 +1,458 @@
+---
+date: '2022-10-19'
+title: "Java Flow Code"
+menu:
+  corda-5-dev-preview2:
+    parent: corda-5-dev-preview-flow
+    identifier: corda-5-dev-preview-flow-java
+    weight: 2000
+section_menu: corda-5-dev-preview2
+---
+
+The Java code for the flows and supporting classes can be found in the CSDE repo at `src/main/java/com/r3/developers/`.
+
+## Java Flow Files
+
+The `src/main/java/com/r3/developers/` folder contains the following:
+* [Message.java](#message)
+* [MyFirstFlow.java](#myfirstflow)
+* [MyFirstFlowResponder.java](myfirstflowresponder)
+* [MyFirstFlowStartArg.java](myfirstflowstartarg)
+
+### Message
```

Comment:
I think might be clearer if the titles were the complete name so "Message.java" instead of "Message".
Similarly for MyFirstFlow, MyFirstFlowResponder and MyFirstFlowStartArgs